cloudyweb/urls.py

```diff
@@ -33,5 +35,5 @@
     url(r'^admin/', include(admin.site.urls)),
     url(r'^api-auth/', include('rest_framework.urls', 
         namespace='rest_framework')),
-    url(r'^api-token-auth/', accounts_views.obtain_auth_token, name='token-auth'),
-]
+    url(r'^api-token-auth/', authtoken_views.obtain_auth_token, name='token-auth'),
```

Comment:
The test has caught a bug, you should be more careful when removing code next time.
Kindly use `accounts_views` instead, since we decided to randomize the token.